tests/test_retrievers_langchain_pathway.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -x
+
+WORKPATH=$(dirname "$PWD")
+LOG_PATH="$WORKPATH/tests"
+ip_address=$(hostname -I | awk '{print $1}')
+function build_docker_images() {
+    cd $WORKPATH
+
+    cd comps/vectorstores/langchain/pathway
+
+    docker build --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -t vectorstore-pathway .
+
+    cd $WORKPATH
+    # --no-cache
+    docker build -t opea/retriever-pathway:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/retrievers/langchain/pathway/docker/Dockerfile .
```

Comment:
Please use `--no-cache` in the test, and use comps as image tag, `opea/retriever-pathway:comps`.